commands/src/main/java/org/jclouds/karaf/commands/table/internal/OSGiScriptEngine.java

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
Regarding licensing, according to [this](http://www.apache.org/legal/src-headers.html#3party) and [this](http://www.apache.org/legal/src-headers.html#notice), we should be good to go. Although it is not the right header for software developed at Apache (that is a Felix issue), we should keep it as-is. No changes to the notice file should be required as the copyright owner of the copied files is already the ASF.